cime_config/namelist_definition_drv.xml

```diff
@@ -233,17 +233,15 @@
     </values>
   </entry>
 
-  <!-- =========================================== -->
-  <!-- All components orbital attributes           -->
-  <!-- =========================================== -->
-
+  <!-- not yet fully implemented, set to false -->
   <entry id="write_restart_at_endofrun">
     <type>logical</type>
     <category>nuopc</category>
     <group>ALLCOMP_attributes</group>
     <values>
       <value>.false.</value>
-      <value rest_option="end">.true.</value>
+      <value rest_option='none'>.false.</value>
+      <value rest_option='never'>.false.</value>
```

Comment:
Why did the rest_option="end" get removed?

Reply:
Because write_restart_at_endofrun still isn't working correctly.   I will address this in an upcoming PR.